.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
+.venv
+result.png
+checklist.md
+.gitignore
+.github
+.idea
+docker-compose.yml
```

Comment:
Ensure docker-compose.yml is not required in the Docker image. It's typically used for orchestration rather than being part of the image itself.